modules.d/00systemd/module-setup.sh

```diff
@@ -186,33 +172,27 @@ install() {
             /etc/sysctl.conf \
             /etc/udev/udev.conf
 
-        mapfile -t _mods < <(modules_load_get /etc/modules-load.d)
-        [[ ${#_mods[@]} -gt 0 ]] && hostonly='' instmods "${_mods[@]}"
+        modules_load_get /etc/modules-load.d
     fi
 
-    if ! [[ -e "$initdir/etc/machine-id" ]]; then
+    if ! [ -e "$initdir/etc/machine-id" ]; then
         : > "$initdir/etc/machine-id"
         chmod 444 "$initdir/etc/machine-id"
     fi
 
     # install adm user/group for journald
+    # we don't use systemd-networkd, but the user is in systemd.conf tmpfiles snippet
     inst_multiple nologin
-    {
-        grep '^systemd-journal:' "$dracutsysrootdir"/etc/passwd 2> /dev/null
-        grep '^adm:' "$dracutsysrootdir"/etc/passwd 2> /dev/null
-        # we don't use systemd-networkd, but the user is in systemd.conf tmpfiles snippet
```

Comment:
I think it is useful to keep the comment.

Reply:
The comment moved above the greps:
```sh
    # install adm user/group for journald
    # we don't use systemd-networkd, but the user is in systemd.conf tmpfiles snippet
    inst_multiple nologin
    grep -e '^systemd-journal:' \
         -e '^adm:' \
         -e '^systemd-network:' "$dracutsysrootdir"/etc/passwd >> "$initdir/etc/passwd" 2> /dev/null
    grep -e '^systemd-journal:' \
         -e '^wheel:' \
         -e '^adm:' \
         -e '^utmp:' \
         -e '^root:' \
         -e '^systemd-network:' "$dracutsysrootdir"/etc/group >> "$initdir/etc/group" 2> /dev/null
```